src/live-share/test/live-share-test

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
```

Comment:
Since the `node_modules/vscode/bin/test` script accepts the path of the tests folder as an ENV variable and the path should be absolute, I've created this file to build up that variable.

Reply:
I'm not sure I understand why there is a `live-share-test` file here (no extension)

Reply:
Your main concern is the absence of the file extension or the presence of the file itself? I agree that the file should have an extension.

Reply:
i see 2 files. one with an extension and one without an extension. im not sure why they both are here. can you clarify?
![image](https://user-images.githubusercontent.com/1202528/58662339-9bfa8200-8332-11e9-8fb9-14820d05bb81.png)

Reply:
it looks like you delete the extra file. thanks